.travis.yml

```diff
@@ -1,25 +1,101 @@
 language: c
-os:
-  - osx
-  - linux
- 
-sudo:
-  - true
 
-compiler:
-- gcc
-- clang
+matrix:
+  include:
+    - os: osx
+      osx_image: xcode10.1
+      compiler: gcc
+      addons:
+        homebrew:
+          packages:
+            autoconf
+            libtool
+            tesseract
+            leptonica
+      script:
+        - cd mac
+        - ./build.command
+        - ./ccextractor --version
 
-before_install:
-- if [[ $TRAVIS_OS_NAME == 'osx' ]]; then brew upgrade automake; brew install pkg-config autoconf automake libtool tesseract leptonica; fi
-- if [[ $TRAVIS_OS_NAME == 'linux' ]]; then sudo apt-get install -y libcurl4-gnutls-dev tesseract-ocr tesseract-ocr-dev libleptonica-dev autoconf-archive; fi
-- if [[ $TRAVIS_OS_NAME == 'linux' ]]; then sudo apt-get install python-distutils-extra tesseract-ocr tesseract-ocr-eng libopencv-dev libtesseract-dev libleptonica-dev python-all-dev swig libcv-dev python-opencv python-numpy python-setuptools build-essential subversion; fi
-- if [[ $TRAVIS_OS_NAME == 'linux' ]]; then wget https://github.com/DanBloomberg/leptonica/releases/download/1.74.4/leptonica-1.74.4.tar.gz && tar xvf leptonica-1.74.4.tar.gz; cd leptonica-1.74.4; ./configure && make && sudo make install; cd ..; fi
-- if [[ $TRAVIS_OS_NAME == 'linux' ]]; then git clone https://github.com/tesseract-ocr/tesseract.git; cd tesseract; ./autogen.sh; ./configure --enable-debug; LDFLAGS="-L/usr/local/lib" CFLAGS="-I/usr/local/include" make; sudo make install; sudo ldconfig; cd ..; fi
+    - os: osx
+      osx_image: xcode10.1
+      compiler: clang
+      addons:
```

Comment:
It looks like you have a lot of whitespace after the end of line... Care to correct? :)

Reply:
Hi! I would love to correct it, but am unable to locate the extra whitespaces. How do I fix it?

![screenshot 2019-01-12 at 7 38 02 pm](https://user-images.githubusercontent.com/12415700/51074162-a7118600-16a1-11e9-838d-ab8801a922aa.png)

Reply:
There's none -_-

It's because the line on the left side is wrapping over multiple ones that this side is as well... My bad.

Reply:
Ah, let me know if any changes are needed 🙂